lib/Driver/Driver.cpp

```diff
@@ -2786,6 +2789,41 @@ void Driver::chooseSwiftModuleOutputPath(Compilation &C,
   }
 }
 
+void Driver::choosePrivateOutputFilePath(Compilation &C,
```

Comment:
Nitpick: can you put "module" in the name of this function somewhere? I don't know if all private outputs belong in the module folder.

Reply:
Alternately, is it possible to combine this with the swiftdoc helper and just have a flag to say "Private" or not? Duplicated code is code that gets out of sync.

Reply:
Sounds good! Will do.